shared/enclave/README.md

```diff
@@ -0,0 +1,34 @@
+# ETH Enclave
+
+Enclave is the communication layer between hardware wallets and the Electron
+web view. This layer is necessary if you've disabled node integration, and
+enabled context isolation on your webview ([Which is something you should do.](https://github.com/electron/electron/blob/master/docs/tutorial/security.md))
+
+Enclave uses Electron's Protocol API to open up an HTTP-like communication layer
```

Comment:
I think request-response would be better in lieu of HTTP here to keep it more generalized

Reply:
Do you mean just calling it that in docs, or restructuring this? It uses things like HTTP methods and HTTP status codes, so it's pretty grounded in HTTP.

Reply:
Just calling it in the docs. Since this could technically use a number of transport layers like when we were using ipcRenderer and postMessage